pkg/types/powervs/defaults/platform.go

```diff
@@ -1,16 +1,27 @@
 package defaults
 
 import (
+	"crypto/rand"
```

Comment:
Shouldn't a pseudo-random number from `math/rand` be enough here?

Reply:
The reason why I went with crypto/rand was because of the following error from lint:
Use of weak random number generator (math/rand instead of crypto/rand) (gosec)

Reply:
Ok. If you decide to go back to `math/rand` it's totally fine to ignore the linter's comment since this is not security-sensitive: `//nolint:gosec <reason here>`. Otherwise, `crypto/rand` is also fine and there should be negligible performance impact.

Reply:
Thanks for the ignore tip!  I'll leave it as it is for now :)